cores/ckan_portal/conf/schema.xml

```diff
@@ -74,39 +104,37 @@
     <field name="id" type="string" indexed="true" stored="true" required="true" />
     <field name="site_id" type="string" indexed="true" stored="true" required="true" />
     <field name="title" type="text" indexed="true" stored="true" />
+    <field name="title_ngram" type="text_ngram" indexed="true" stored="true" />
```

Comment:
are we storing the title_ngram for debugging purposes?

Reply:
So looks like `ngram` in Solr is for better text matching? But there are apparently some performance issues using ngrams, so I don't think we really need them eh @thriuin ?

Reply:
Unless if we are using `ngram` fields in Django Searches? Might as well use it in CKAN too.

Reply:
I am not sure why you would use a ngram with a title field like this - I thought this was more for fixed width code strings. I am not using ngrams in Search so I think you could drop it